AppCenter/AppCenter/Internals/Context/AuthToken/MSAuthTokenContext.h

```diff
@@ -10,11 +10,6 @@ NS_ASSUME_NONNULL_BEGIN
  */
 @interface MSAuthTokenContext : NSObject
 
-/**
- * Auth token.
- */
-@property(nonatomic, nullable) NSString *authToken;
```

Comment:
What's the reason of the change? `atomic` isn't enough?

Reply:
@jaelim-ms `authToken` is changed from a property to our own getter/setter methods.

Reply:
My original question was "why do you need to cumstomize getter/setter for the property?".
The getter only synchronize the property, there is no customer setter.
You can keep this line (and remove the property declaration in .m file) and change the property to atomic will do exactly same thing that you changed here.